tools/list_plugins

```diff
@@ -0,0 +1,64 @@
+#! /usr/bin/env python
+from itertools import chain
+import os.path
+import sys
+import xml.etree.ElementTree as ET
+
+import roslib
+import rospkg
+
+INDENT = "  "
+
+
+def list_plugins():
```

Comment:
Add some docstrings to explain what this does. I initially thought this would show all active plugins on an instance of ed. But it lists the plugin types known to the system.